core/service/src/backup/custodian.rs

```diff
@@ -23,12 +25,74 @@ use threshold_fhe::{execution::runtime::party::Role, hashing::DomainSep};
 use super::{
     error::BackupError,
     operator::{BackupMaterial, InnerOperatorBackupOutput},
-    traits::{BackupDecryptor, BackupSigner},
 };
 
 pub(crate) const HEADER: &str = "ZAMA TKMS SETUP TEST OPERATORS-CUSTODIAN";
 pub(crate) const DSEP_BACKUP_CUSTODIAN: DomainSep = *b"BKUPCUST";
 
+#[derive(Clone, Serialize, Deserialize, VersionsDispatch)]
+pub enum InternalCustodianRecoveryOutputVersioned {
+    V0(InternalCustodianRecoveryOutput),
+}
+
+/// This is the message that a custodian sends to an operator after starting recovery.
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, Versionize)]
+#[versionize(InternalCustodianRecoveryOutputVersioned)]
+pub struct InternalCustodianRecoveryOutput {
```

Comment:
should we add backward compatibility tests for these? they'll need to go into the xxx-v0.12 files though